terraformer_sg.tf

```diff
@@ -0,0 +1,113 @@
+# Security group for Terraformer instances
+resource "aws_security_group" "terraformer" {
+  provider = aws.provisionassessment
+
+  vpc_id = aws_vpc.assessment.id
+
+  tags = {
+    Name = "Terraformer"
+  }
+}
+
+# Allow egress anywhere via ssh
+#
+# For: Terraformer instances must be able to configure redirectors and
+# operations instances via Ansible.
+resource "aws_security_group_rule" "terraformer_egress_anywhere_via_ssh" {
+  provider = aws.provisionassessment
+
+  security_group_id = aws_security_group.terraformer.id
+  type              = "egress"
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  from_port         = 22
+  to_port           = 22
+}
+
+# Allow egress anywhere via port 5986 (Windows Remote Manager).
+#
+# For: Terraformer instances must be able to configure Windows-based
+# operations instances via Ansible.
+resource "aws_security_group_rule" "terraformer_egress_to_operations_via_winrm" {
+  provider = aws.provisionassessment
+
+  security_group_id = aws_security_group.terraformer.id
+  type              = "egress"
+  protocol          = "tcp"
+  cidr_blocks       = [aws_subnet.operations.cidr_block]
+  from_port         = 5986
+  to_port           = 5986
+}
```

Comment:
Is there a reason this block is not at the bottom of this file?  It breaks the order of our rules being sorted by port number.

Reply:
I was thinking that the `ssh` and `winrm` jazz should live together since it serves the same purpose.  You're argument is just as valid, though, so I'll move it.

Reply:
See commit 200ce1b.

Reply:
I knew there was some method to your madness, but I wasn't sure what it was.  Thanks for moving that block. 👍